mmv1/templates/terraform/examples/looker_instance_full.tf.erb

```diff
@@ -0,0 +1,48 @@
+resource "google_looker_instance" "<%= ctx[:primary_resource_id] %>" {
+  name               = "<%= ctx[:vars]["instance_name"] %>"
+  platform_edition   = "LOOKER_CORE_STANDARD"
+  region             = "us-central1"
+  private_ip_enabled = false
+  public_ip_enabled  = true
+  reserved_range     = ""
```

Comment:
Can we supply nonempty values for this and `consumer_network`? We can create other resources inline in the config if needed.

Reply:
Done.